static/templates/compose_control_buttons.hbs

```diff
@@ -1,15 +1,18 @@
 <div class="compose_control_buttons_container order-1">
-    <input type="file" class="file_input notvisible pull-left" multiple />
-    {{#if file_upload_enabled }}
-    <a role="button" class="compose_control_button compose_upload_file fa fa-paperclip notdisplayed" aria-label="{{t 'Upload files' }}" tabindex=0 data-tippy-content="{{t 'Upload files' }}"></a>
-    {{/if}}
     <a role="button" class="markdown_preview compose_control_button fa fa-eye" aria-label="{{t 'Preview' }}" tabindex=0 data-tippy-content="{{t 'Preview' }}"></a>
     <a role="button" class="undo_markdown_preview compose_control_button fa fa-edit" aria-label="{{t 'Write' }}" tabindex=0 style="display:none;" data-tippy-content="{{t 'Write' }}"></a>
-    <a role="button" class="compose_control_button fa fa-video-camera video_link" aria-label="{{t 'Add video call' }}" tabindex=0 data-tippy-content="{{t 'Add video call' }}"></a>
-    <a role="button" class="compose_control_button fa fa-smile-o emoji_map" aria-label="{{t 'Add emoji' }}" tabindex=0 data-tippy-content="{{t 'Add emoji' }}"></a>
-    <a role="button" class="compose_control_button fa fa-clock-o time_pick" aria-label="{{t 'Add global time' }}" tabindex=0 data-tooltip-template-id="add-global-time-tooltip" data-tippy-maxWidth="none"></a>
-    <a role="button" class="compose_control_button compose_gif_icon {{#unless giphy_enabled }} hide {{/unless}} zulip-icon zulip-icon-gif" aria-label="{{t 'Add GIF' }}" tabindex=0 data-tippy-content="{{t 'Add GIF' }}"></a>
-    <div class="divider hide-sm">|</div>
+    <div class="compose_control_buttons_container preview_mode_disabled">
```

Comment:
Is it intended that this generates a setup where we have a `compose_control_buttons_container` inside another?

Reply:
I guess this is the CSS that will be applied twice:
```
.compose_control_buttons_container {                                                                     
    margin-right: auto;                                                                                  
    display: flex;                                                                                       
    gap: 4px;                                                                                            
    align-items: center;                                                                                 
```

Reply:
It seems possible it'd be cleaner to just have the `preview_mode_disabled` attribute on each individual element below, rather than adding this `div`.  But I'm happy with any clean solution here.

Reply:
> Is it intended that this generates a setup where we have a `compose_control_buttons_container` inside another?

Yes, It is because icons inside that div don't look weird.

Reply:
To disable buttons properly we have to use that extra div.
```
.preview_mode_disabled {
        cursor: not-allowed;
        opacity: 0.3;

        .compose_control_button {
            pointer-events: none;
        }
    }
```
This CSS can't be applied to the same element, It has to be parent and child.

Reply:
@timabbott your thoughts?